funnel/views/helpers.py

```diff
@@ -20,3 +22,9 @@ def localize_date(date, from_tz=utc, to_tz=utc):
             to_tz = pytz_timezone(to_tz)
         return from_tz.localize(date).astimezone(to_tz).replace(tzinfo=None)
     return date
```

Comment:
These methods should be in Coaster. Where are they being used?

Reply:
@iambibhas Could you please look into this. It's used in schedule and session views.

Reply:
Along the 3, only `localize_date()` is used throughout funnel. Rest 2 are not used anywhere. I can add it to coaster.